content/hardware/08.edu/solution-and-kits/alvik/tutorials/getting-started/getting-started.md

```diff
@@ -108,6 +125,6 @@ You are now all set, disconnect Alvik from the computer, put some obstacles arou
 
 ### Next Steps
 
-* There is a set of already built examples that will help you to better understand how Alvik works, you can download them from [this link](https://github.com/arduino/arduino-alvik-mpy/archive/refs/tags/0.2.0.zip), unzip them in your already created _alvik_ folder and you will be able to see them straight away in the Arduino Labs for MicroPython IDE.
```

Comment:
Is there a specific reason why this particular version is used? This looks like a old version https://github.com/arduino/arduino-alvik-mpy/releases

Reply:
@g-castronovo If new versions are compatible we might benefit from not linking directly the download and instead send to the release pages as new versions will render the content outdated :) Will still work but they will be on a different verison